README.md

```diff
@@ -1,14 +1,28 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
 ## Latest Release: v2.12
+
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md)
 
 [Release Notes for past versions](release-notes.md)
 
-## Working Changes
+## SDLC
+
+### Working Changes
```

Comment:
Anything on Master should be ready to go out in a published version, so Master is our release-candidate branch. No?